core/src/main/java/org/pentaho/di/connections/ConnectionManager.java

```diff
@@ -706,12 +677,25 @@ public void copy( IMetaStore sourceMetaStore, IMetaStore destinationMetaStore )
    * @return A list of value/label pairs of named connection types
    */
   public List<Type> getItems() {
+    return getItemsByType( null );
+  }
+
+  /**
```

Comment:
Added to support `VFSConnectionManagerWrapper#getItems()` that filters for `VFSConnectionProvider`.